sylt/src/formatter.rs

```diff
@@ -509,33 +544,29 @@ fn write_statement<W: Write>(dest: &mut W, indent: u32, statement: Statement) ->
             }
         }
     }
+    write!(dest, "\n")?;
 
     Ok(())
 }
 
 /// Replace consecutive empty statements with one empty statement with all comments of the previous statements.
-//TODO(gu): Rewrite the formatter to use moves instead of borrows. Then we wouldn't need to clone when passing
-//          into this function.
-fn merge_empty_statements(mut statements: Vec<Statement>) -> Vec<Statement> {
-    // Reverse since
-    // - we always want to remove and look at the first statement and
-    // - pop() is faster than remove(0).
-    statements.reverse();
-
-    let mut ret = Vec::new();
-    while let Some(mut statement) = statements.pop() {
-        // Begin eating empty statements
-        while matches!(
-            statements.last().map(|s| &s.kind),
-            Some(StatementKind::EmptyStatement)
-        ) {
-            statement
-                .comments
-                .append(&mut statements.pop().unwrap().comments);
-        }
-        ret.push(statement);
-    }
-    ret
+// TODO(gu): Rewrite the formatter to use moves instead of borrows. Then we wouldn't need to clone when passing
```

Comment:
@sornas

Reply:
I see. You'll find that all my todos are written this way.

Reply:
Fixed 'em for you ;)